components/automate-ui/src/app/pages/integrations/list/integrations-list.component.html

```diff
@@ -54,10 +54,10 @@ <h2 class="display4 strong">Cloud Platform Providers</h2>
             <chef-td class="column-date">{{ manager.date_added | datetime: RFC2822 }}</chef-td>
             <chef-td>{{ manager.status }}</chef-td>
             <chef-td class="controls">
```

Comment:
Not your doing, but would you mind cleaning up this errant item to use `three-dot-column` class to make consistency complete?